package.json

```diff
@@ -2,26 +2,41 @@
   "name": "sgid-test-app",
   "version": "2.0.0",
   "description": "This is an example node application that implements sgID's OAuth2 API.",
-  "main": "index.js",
+  "main": "dist/index.js",
   "engines": {
     "node": ">=16.16"
   },
   "dependencies": {
-    "@opengovsg/sgid-client": "^1.0.4",
+    "@opengovsg/sgid-client": "^2.0.0",
+    "cookie-parser": "^1.4.6",
     "dotenv": "8.6.0",
     "ejs": "3.1.7",
     "express": "4.17.1",
     "helmet": "4.6.0",
     "jsonwebtoken": "8.5.1",
+    "node-cache": "^5.1.2",
     "node-fetch": "2.6.7",
-    "node-jose": "2.1.1"
+    "node-jose": "2.1.1",
+    "uuid": "^9.0.0"
   },
   "devDependencies": {
-    "nodemon": "^2.0.6"
+    "@swc/core": "^1.3.60",
+    "@swc/helpers": "^0.5.1",
+    "@types/cookie-parser": "^1.4.3",
+    "@types/express": "^4.17.17",
+    "@types/node": "^20.1.7",
+    "@types/node-cache": "^4.2.5",
+    "@types/uuid": "^9.0.1",
+    "nodemon": "^2.0.6",
+    "regenerator-runtime": "^0.13.11",
```

Comment:
may i know why we need the `regenerator-runtime` package? i also don't see it imported in the codebase

Reply:
It is used by swc